### ISSUE TITLE
Display formatter: field instances should be distinct

### ISSUE THREAD

**Issue by herbdool** [backdrop-contrib/paragraphs_jquery_ui_accordion#17]
Right now the setting are combining the field instances. If paragraphs are sharing fields then it'll show the field title on either one or the other - it won't be clear if you've got the right one.

Ideally choosing the paragraph bundle would filter the field options. But at the very least I think each field instance should be listed separately. And maybe put the machine name as well.

**Comment by herbdool** [comment]
@yorkshire-pudding I've got a PR. I also tried the ajax approach of dependent fields - bundle and bundle instances. I got close but paused because it's not fully working. It could be more difficult because the settings form is already populated via ajax. Or maybe I'm overlooking something simple.

**Comment by herbdool** [comment]
Here's my alternative branch: https://github.com/herbdool/paragraphs_jquery_ui_accordion/tree/issue-17b with the ajax. I gave up before I got it all working.

**Comment by yorkshire-pudding** [comment]
Hi @herbdool - I have to say I don't use this module much now I have the other module.  The Readme does suggest just use 1 paragraph type but I understood it does work with multiple paragraph types if the fields are shared, though the limitations of this module means it should be ok to limit to a single accordion type, but it could count as a breaking change if people _are_ using with multiple types.

It would be good to also get @alanmels 's views on this.

I have to say, I haven't had much luck with dependent dropdown fields before; my latest attempt was in a node edit form field, but I couldn't get it to work.

The display settings seems to ignore completely which paragraph bundles are actually enabled for that field; I tested with one paragraph type enabled but it still gives the list.

I'm not sure the PR as it stands improves matters as while it gives lots of information, it actually looks more complex now.

**Comment by herbdool** [comment]
It may look more complex, but currently it hides important information if fields are shared but have different labels.

I have gotten dependent fields working, just not in this case.

**Comment by alanmels** [comment]
Hi All, we use this module on multiple websites and I'm open to any kind of improvements as long as existing websites relying on the module don't break. Unfortunately, no time for diving into this myself.

**Comment by yorkshire-pudding** [comment]
As per comment in Zulip (https://backdrop.zulipchat.com/#narrow/stream/218635-Backdrop/topic/Paragraphs.20Accordion/near/425606262), closing this as OP not progressing this.
